doc/articles/common-issues-ios.md

```diff
@@ -1,18 +1,18 @@
 ---
-uid: Uno.UI.CommonIssues.IosCatalyst
+uid: Uno.UI.CommonIssues.Ios
 ---
 
-# Issues related to iOS/Mac Catalyst projects
+# Issues related to iOS projects
 
 ## Developing on older Mac hardware
 
-The latest macOS release and Xcode version are required to develop with Uno Platform for iOS and Mac Catalyst. However, if you have an older Mac that does not support the latest macOS release, you can use a third-party tool to upgrade it such as [OpenCore Legacy Patcher](https://dortania.github.io/OpenCore-Legacy-Patcher/). While not ideal, this can extend the use of older hardware by installing the latest macOS release on it. Please note that this method is not required when developing for other targets such as Android, Skia, WebAssembly, or Windows.
+The latest macOS release and Xcode version are required to develop with Uno Platform for iOS. However, if you have an older Mac that does not support the latest macOS release, you can use a third-party tool to upgrade it such as [OpenCore Legacy Patcher](https://dortania.github.io/OpenCore-Legacy-Patcher/). While not ideal, this can extend the use of older hardware by installing the latest macOS release on it. Please note that this method is not required when developing for other targets such as Android, Skia, WebAssembly, or Windows.
```

Comment:
```suggestion
The latest macOS release and Xcode version are required to develop with Uno Platform for iOS. However, if you have an older Mac that does not support the latest macOS release, you can use a third-party tool to upgrade it, such as [OpenCore Legacy Patcher](https://dortania.github.io/OpenCore-Legacy-Patcher/). While not ideal, this can extend the use of older hardware by installing the latest macOS release on it. Please note that this method is not required when developing for other targets such as Android, Skia, WebAssembly, or Windows.
```